config/services/proxy.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>proxy</short>
+  <description>HTTP/HTTPS proxy server</description>
+  <port protocol="tcp" port="3128"/>
```

Comment:
Port `3128` belongs to `ndl-aas` according to ICANN. Furthermore the service name `proxy` is too generic.

Where are you getting port `3128` ?

Reply:
1 Port 3128 is used by default by all proxy servers I know. 
2 I suggested other options in #1389, such as proxy-http and proxy-socks, and I'm fine with those too.
3 https://github.com/firewalld/firewalld/blob/main/config/services/squid.xml This in no way prevented you from adding the squid service with an even worse name, even though squid is far from being the only proxy server.

Reply:
I'd like to see it renamed to `proxy-http`.

Reply:
`socks` is okay. That's the IANA assigned name.